cookbooks/bach_krb5/recipes/gems.rb

```diff
@@ -0,0 +1,32 @@
+require 'pathname'
+require 'rubygems'
+gem_path = Pathname.new(Gem.ruby).dirname.join("gem").to_s
+
+gem_package "rake-compiler" do
+  gem_binary gem_path
+  version ">=0.0.0"
+  action :nothing
+end.run_action(:install)
+
+node['krb5']['devel']['packages'].each do |pkg|
```

Comment:
Yes, this parameter is defined in upstream cookbook `krb5` and `bach_krb5` depends on it.